docs/data-sources/environment.md

```diff
@@ -0,0 +1,38 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bytebase_environment Data Source - terraform-provider-bytebase"
+subcategory: ""
+description: |-
+  The environment data source.
+---
+
+# bytebase_environment (Data Source)
+
+The environment data source. You can get environment through `bytebase_environment` data source.
```

Comment:
The environment data source. You can get a single environment through `bytebase_environment` data source.